generators/express/templates/base/src/infrastructure/database/repositories/UserRepository.ts

```diff
@@ -19,6 +19,10 @@ class DatabaseUserRepository implements UserRepository {
     findByEmail(email: string): Promise<User | null> {
         return this.#dataSource.manager.findOneBy(User, { email });
     }
+
+    save(user: User): Promise<void> {
+        return this.#dataSource.manager.save(user);
+    }
```

Comment:
```suggestion
    async save(user: User): Promise<void> {
        await this.#dataSource.manager.save(user);
    }
```
Ça devrais fix la CI.